### ISSUE TITLE
[ BUG ] `Uninstall-FalconSensor` fails on some Windows machines because of multiple uninstallers available in the registry

### ISSUE THREAD

**Issue by dvd00** [CrowdStrike/psfalcon#317]
**Describe the bug**
I'm not sure why, but in some instances there were two entries in the Windows registry for the sensor uninstallation.
![image](https://github.com/CrowdStrike/psfalcon/assets/22195644/493830ba-d5bf-4143-a7ed-62555969d6f3)
![image](https://github.com/CrowdStrike/psfalcon/assets/22195644/e2c12f12-cb08-4b67-b388-d15f39441376)

When the uninstall script tries to retrieve the uninstall command from the registry, it complains about some strange type casting error. Actually, it finds two keys, and one of them needs to be chosen before it can do `.GetValue("QuietUninstallString")`.
(The error is about trying to call .GetValue with a string parameter on a list. Sorry I forgot to take a screenshot about that)

**To Reproduce**
I observed this behaviour on different machine configurations. Maybe, it's something related to an incorrect installation process.

**Expected behavior**
I think that just choosing one of the two would be fine, isn't it?

**Environment (please complete the following information):**
 - OS: Windows
 - PSFalcon: 2.2.5

**Additional context**
I was able to uninstall the sensor after changing this line of the script in psf-sensors.ps1
```powershell
# OLD LINE
$UninstallString = $RegKey.GetValue("QuietUninstallString")
# NEW LINE
$UninstallString = ($RegKey | Select-Object -First 1).GetValue("QuietUninstallString") 
```

I'm not sure if this is the right solution and I'm not good at Powershell. I just like to share what worked for me.

**Comment by bk-cs** [comment]
Thank you for the report!

I ran into a similar issue while working on something else. I think I'll update the command to check for multiple versions in the registry, and if they're found, remove the version that's currently installed by checking the version of the executable that's present in the relevant program folder.

I'll work on this soon and let you know when I have an updated version to test.

**Comment by bk-cs** [comment]
I created a new version of `Uninstall-FalconSensor` using a new script that will only attempt to uninstall the version that's found within the Program Files directory. Would you be willing to test? You can update your `Uninstall-FalconSensor` command by updating the corresponding file, reloading PowerShell and re-importing PSFalcon.

If using PowerShell 5.1:
```powershell
(Invoke-WebRequest https://raw.githubusercontent.com/CrowdStrike/psfalcon/master/public/psf-sensors.ps1).Content > $HOME\WindowsPowerShell\Modules\PSFalcon\2.2.5\public\psf-sensors.ps1
```

**Comment by bk-cs** [comment]
I did mean 2.2.6; thank you. Corrected my example.

**Comment by bk-cs** [comment]
Resolved in 2.2.6 release which has been published to GitHub and will be published on the PowerShell Gallery soon.
